packages/core/src/main/conversation/ConversationService.ts

```diff
@@ -108,16 +107,27 @@ class ConversationService {
     return genericMessage;
   }
 
-  // TODO: The correct functionality of this function is heavily based on the case that it always runs into the catch
-  // block
-  private getPreKeyBundles(conversationId: string): Promise<ClientMismatch | UserPreKeyBundleMap> {
-    return this.apiClient.conversation.api.postOTRMessage(this.clientID, conversationId).catch((error: AxiosError) => {
-      if (error.response && error.response.status === 412) {
-        const recipients: UserClients = error.response.data.missing;
-        return this.apiClient.user.api.postMultiPreKeyBundles(recipients);
+  private async getPreKeyBundles(
```

Comment:
Since `generatePreKeyBundles` has been renamend to `generatePreKeyBundle`, the `getPreKeyBundles` function should be renamed too into something like `getPreKeyBundle`?

Reply:
The renaming just happened in a test file. I'm not sure if we should rename this function.